storybook/pages/WalletAccountListItemPage.qml

```diff
@@ -20,7 +20,7 @@ SplitView {
             emoji: emojiField.text
             walletColor: walletColorField.text
             currencyBalance: QtObject {
-                readonly property double amount: parseInt(currencyBalanceField.text)
+                readonly property double amount: parseFloat(currencyBalanceField.text)
```

Comment:
didnt we decide not wanting to use doubles to store balances?

Reply:
Yeah sure, this is just to parse the text from the input field in storybook